backend/app/modules/user/antisybil/service/initial.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+import time
+from typing import List
+
+from app.extensions import db
+from app.context.manager import Context
+from app.infrastructure import database
+from app.pydantic import Model
+from app.infrastructure.external_api.gc_passport.score import (
+    issue_address_for_scoring,
+    fetch_score,
+    fetch_stamps,
+)
+
+
+class InitialUserAntisybil(Model):
+    def get_antisybil_status(self, _: Context, user_address: str) -> (int, datetime):
+        score = database.user_antisybil.get_score_by_address(user_address)
+        return score.score, score.expires_at
+
+    def fetch_antisybil_status(
+        self, _: Context, user_address: str
+    ) -> (str, datetime, any):
+        score = issue_address_for_scoring(user_address)
+
+        while score["status"] != "DONE":
```

Comment:
Please create proper function for refetching in the external_api package. You can take a look at ```retry_request``` that currently implements this logic and try to unify it. 

Besides that we should consider:
1. Whether backend should do ```time.sleep``` or it's rather FE-related stuff
2. Move "DONE" value to a separate enum class with statuses for Gitcoin API.

Reply:
I've reused retry_request, moved it into "common".

As long as GP is fast-ish (don't exceed default browser http timeout) I want to keep API simple.